src/Xamarin.Android.Build.Tasks/Microsoft.Android.Sdk/targets/Microsoft.Android.Sdk.DefaultProperties.targets

```diff
@@ -26,8 +26,6 @@
     <_GetChildProjectCopyToPublishDirectoryItems>false</_GetChildProjectCopyToPublishDirectoryItems>
     <UseMonoRuntime Condition=" '$(UseMonoRuntime)' == '' ">true</UseMonoRuntime>
 
-    <!-- Use $(AndroidMinimumSupportedApiLevel) for $(SupportedOSPlatformVersion) if unset -->
-    <SupportedOSPlatformVersion Condition=" '$(SupportedOSPlatformVersion)' == '' ">$(AndroidMinimumSupportedApiLevel)</SupportedOSPlatformVersion>
```

Comment:
@rolfbjarne from your memory of the meeting, should we actually leave this in and add a similar default in xamarin/xamarin-macios?

Reply:
We talked about doing it either way:

* Switch the default to the minimum across all workloads.
* Not set a default in any SDK workload, and make MAUI/Blazor/whomever set a default value.

Note that the [spec][1] says "Project files may specify a `SupportedOSPlatformVersion` property, and if they do not it will default to OS Version equivalent to the OS API version specified in the `TargetPlatformVersion` value."

I don't know what Google does with Android, but Xcode does more or less the same thing, defaults to the latest OS version.

I think either way can work, although I lean a little bit towards the latter (it's what the spec says, but I might also be influenced by the fact that I wouldn't have to do anything...). I also think the latter way works better if MAUI/Blazor/whomever wants a different default than the min supported by the SDK.

[1]: https://github.com/dotnet/designs/blob/bba3216250cb29b0063bac3ebb57a542ee21ad4f/accepted/2020/minimum-os-version/minimum-os-version.md#msbuild-properties